index.html

```diff
@@ -289,6 +289,16 @@
       <li>In the absence of both HTTP Link headers and embedded link elements, subscribers MUST look in the Host-Meta Well-Known URI [[!RFC6415]] <samp>/.well-known/host-meta</samp> for the &lt;Link&gt; element with rel="hub". However, please note that this mechanism is currently at risk and may be deprecated.</li>
     </ol>
     <p class="warning">Note: The Host-Meta method of discovery is currently At Risk and may be deprecated.</p>
+    <section id="content-negociation">
+      <h3>Content Negociation</h3>
+      <p>
+        For preactical purposes, it is important that the <samp>rel=self</samp> URL only offers a single representation. As the hub has no way of knowing what mime-type may have been requested by the subscriber upon discovery, it would not be able to perform notifications using the same mime-type.
+      </p>
+      <p>
+        It is however possible to perform content negociation (described in Section 12 of [[!RFC2616]]) provided that the publisher yields a <samp>rel=self</samp> that itself offers a single representation.
```

Comment:
"It is, however,"  ...

then change the rest of the sentence starting at "provided that..." to:

"by returning an appropriate <samp>rel=self</samp> URL according to the HTTP headers used in content negotiation. For example, a request to <samp>/feed</samp> with an <samp>Accept</samp> header containing <samp>application/json</samp> could return a <samp>rel=self</samp> value of <samp>/feed.json</samp>."